tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -77,7 +77,7 @@ public function __toString()
 @@ @@
 -<?php namespace Foo\Bar;
 +<?php
- 
```

Comment:
If you need these trailing spaces in the strings, then you can write them like this:
```php
$str = 'Foo
    '.'
Bar';
```
Then the tests will pass but the trailing spaces won't eventually get removed by an IDE that is configured to.